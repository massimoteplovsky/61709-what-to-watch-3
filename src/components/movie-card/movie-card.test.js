import React from 'react';
import renderer from 'react-test-renderer';
import MovieCard from './movie-card.jsx';

const film = {
  id: 1,
  title: `Terminator 2. Judgment day`,
  poster: ``
};

const handleTitleClick = () => {};
const handleMouseEnter = () => {};

it(`<MovieCard /> component renders correctly`, () => {
  const tree = renderer
    .create(<MovieCard
      film={film}
      onTitleClick={handleTitleClick}
      onMouseEnter={handleMouseEnter}
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
