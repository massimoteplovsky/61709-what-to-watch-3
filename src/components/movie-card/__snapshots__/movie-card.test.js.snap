// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieCard /> component renders correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="Terminator 2. Judgment day"
      height="175"
      src=""
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
    onClick={[Function]}
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      Terminator 2. Judgment day
    </a>
  </h3>
</article>
`;
